docs/nv2-sign-verify-aks.md

```diff
@@ -0,0 +1,418 @@
+      --output jsonc
+    ```
+    In the command output, note the `zoneRedundancy` property for the registry. When enabled, the registry is zone redundant, and ORAS Artifact enabled:
+
+    ```JSON
+    {
+      [...]
+      "zoneRedundancy": "Enabled",
+    }
+    ```
+
+## Create an Azure Kubernetes Cluster
+
+If needed, create an Azure Kubernetes Cluster
+
+1. Create an Azure resource group:
+
+    ```azurecli-interactive
+    az group create -n $AKS_RG -l $LOCATION
+    ```
+
+2. Create an AKS cluster with the [az aks create][az-aks-create] command.
+
+    ```azurecli-interactive
+    az aks create -n $AKS_NAME -g $AKS_RG #--attach-acr $ACR_NAME
+    az aks update -n $AKS_NAME -g $AKS_RG --attach-acr $ACR_NAME
+    ```
+
+3. Get the AKS credentials
+
+    ```azurecli-interactive
+    az aks get-credentials -n $AKS_NAME -g $AKS_RG
+    ```
+
+3. Create an ACR Token for notation signing and the ORAS cli to access the registry
+
+    ```azure-cli
+    export NOTATION_USERNAME=$ACR_NAME'-token'
+    export NOTATION_PASSWORD=$(az acr token create -n $NOTATION_USERNAME \
+                        -r $ACR_NAME \
+                        --scope-map _repositories_admin \
+                        --only-show-errors \
+                        -o json | jq -r ".credentials.passwords[0].value")
+    ```
+
+4. Configure permissions for ratify
+    > NOTE: these are temporary steps. Ratify should use the node secrets, using ACR
+    ```bash
+    kubectl create secret docker-registry regcred \
+        --docker-server=$REGISTRY \
+        --docker-username=$NOTATION_USERNAME \
+        --docker-password=$NOTATION_PASSWORD \
+        --docker-email=someone@example.com
+    ```
+
+## Install Gatekeeper
+
+In this step, Gatekeeper will be configured, enabling deployment policies.
+
+1. install Gatekeeper
+
+    ```azurecli-interactive
+    helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+    helm install gatekeeper/gatekeeper  \
+        --name-template=gatekeeper \
+        --namespace gatekeeper-system --create-namespace \
+        --set enableExternalData=true \
+        --set controllerManager.dnsPolicy=ClusterFirst,audit.dnsPolicy=ClusterFirst
+    ````
+ 
+## Build, Validate, Deploy
+
+We're starting with the following basic Azure service configurations
+
+- An ACR with ORAS Artifacts Support, enabling graphs of supply chain artifacts
+- An AKS instance, linked to ACR for pulling images
+- AKS with Gatekeeper support
+
+We'll now demo:
+
+1. Creating a cert in Azure Key Vault, used for remote signing
+   > NOTE: The cert can be a CA issues, or self-signed cert. It's really up to the the user whether they need revocation for their controlled environment scenarios
+2. Secure a namespace with images with trusted signatures
+3. Build an image
+4. Remotely sign the image with Azure Key Vault
+5. Deploy the image to AKS - with a success
+6. Attempt to deploy an unsigned image, and watch it fail.
+
+### Deploy an image
```

Comment:
Should this be `## Deploy an image`? Header 2 instead of Header 3?

Reply:
It was intended to be nested under the Build, Validate, Deploy heading.